addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/InsecureHttpMethodScanRuleUnitTest.java

```diff
@@ -34,6 +38,36 @@ protected InsecureHttpMethodScanRule createScanner() {
         return new InsecureHttpMethodScanRule();
     }
 
+    @ParameterizedTest
+    @CsvSource({"PUT, json", "PUT, xml", "PATCH, json", "PATCH xml"})
```

Comment:
@kingthorin is referring to `"PATCH xml"` which should have the comma after the method.